plugins/emc/emc_vnx_file_

```diff
@@ -0,0 +1,516 @@
+			graph_vlabel uSec / call
+			graph_category nfs
+			graph_args --base 1000
+			EOF
+			for graph in "${graphs[@]}"; do
+				field=$(echo "$graph" | cut -d '.' -f4 )
+				echo "${server}_$field.label $server $field"
+			done
+			cat <<-EOF
+			
+			multigraph vnx_emc_v4_op_percent
+			graph_title EMC VNX 5300 NFSv4 Op %
+			graph_vlabel %
+			graph_scale no
+			graph_category nfs
+			EOF
+			for graph in "${graphs[@]}"; do
+				field=$(echo "$graph" | cut -d '.' -f4 )
+				echo "${server}_$field.label $server $field"
+				echo "${server}_$field.min 0"
+			done
+
+#nfs.client data
+#							 Total    Read     Write   Suspicious   Total    Read     Write      Avg   
+#                                                      Ops/s    Ops/s    Ops/s    Ops diff    KiB/s    KiB/s    KiB/s   uSec/call
+			member_elements=$($SSH server_stats server_2 -monitor nfs.client -count 1 -terminationsummary no -titles never | sed -ne 's/^.*id=//p' | cut -d' ' -f1)
+			readarray graphs2 <<< "$member_elements"
+			cat <<-EOF
+
+			multigraph vnx_emc_nfs_client_ops_s
+			graph_title EMC VNX 5300 NFS Client Ops/s
+			graph_vlabel Ops/s
+			graph_category nfs
+			EOF
+			echo -n "graph_order "
+			for graph in "${graphs2[@]}"; do
+				field=$(echo "$graph" | sed -ne 's/\./_/pg' )
+				echo -n "${server}_${field}_r ${server}_${field}_w ${server}_${field}_t ${server}_${field}_s "
+			done
+			echo " "
+			for graph in "${graphs2[@]}"; do
+				field=$(echo "$graph" | sed -ne 's/\./_/pg' )
+				echo "${server}_${field}_r.label $server $field Read Ops/s"
+				echo "${server}_${field}_w.label $server $field Write Ops/s"
+				echo "${server}_${field}_w.draw STACK"
+				echo "${server}_${field}_t.label $server $field Total Ops/s"
+				echo "${server}_${field}_s.label $server $field Suspicious Ops diff"
+			done
+
+			cat <<-EOF
+
+			multigraph vnx_emc_nfs_client_kib_s
+			graph_title EMC VNX 5300 NFS Client B/s
+			graph_vlabel B/s
+			graph_category nfs
+			EOF
+			echo -n "graph_order "
+			for graph in "${graphs2[@]}"; do
+				field=$(echo "$graph" | sed -ne 's/\./_/pg' )
+				echo -n "${server}_${field}_r ${server}_${field}_w ${server}_${field}_t "
+			done
+			echo " "
+			for graph in "${graphs2[@]}"; do
+				field=$(echo "$graph" | sed -ne 's/\./_/pg' )
+				echo "${server}_${field}_r.label $server $field Read B/s"
+				echo "${server}_${field}_r.cdef ${server}_${field}_r,1024,*"
+				echo "${server}_${field}_w.label $server $field Write B/s"
+				echo "${server}_${field}_w.cdef ${server}_${field}_w,1024,*"
+				echo "${server}_${field}_w.draw STACK"
+				echo "${server}_${field}_t.label $server $field Total B/s"
+				echo "${server}_${field}_t.cdef ${server}_${field}_t,1024,*"
+			done
+
+			cat <<-EOF
+
+			multigraph vnx_emc_nfs_client_avg_usec
+			graph_title EMC VNX 5300 NFS Client Avg uSec/call
+			graph_vlabel uSec/call
+			graph_category nfs
+			EOF
+			for graph in "${graphs2[@]}"; do
+				field=$(echo "$graph" | sed -ne 's/\./_/pg' )
```

Comment:
Regarding dots: isn't this the point of your `sed` filter above?

In general I would suggest, to use `clean_fieldname` whenever you are not sure (i.e. the value is not safe by specification). A lot of issue tickets are related to invalid fieldname cleanup, even though this can be easily avoided.